test/ExportQueryExecutionTreesTest.cpp

```diff
@@ -1667,3 +1667,109 @@ TEST(ExportQueryExecutionTrees, convertGeneratorForChunkedTransfer) {
               AllOf(HasSubstr("!!!!>># An error has occurred"),
                     HasSubstr("A very strange")));
 }
+
+TEST(ExportQueryExecutionTrees, idToLiteralOrIriFunctionality) {
+  std::string kg =
+      "<s> <p> \"something\" . <s> <p> 1 . <s> <p> "
+      "\"some\"^^<http://www.w3.org/2001/XMLSchema#string> . <s> <p> "
+      "\"dadudeldu\"^^<http://www.dadudeldu.com/NoSuchDatatype> .";
+  auto qec = ad_utility::testing::getQec(kg);
+  auto getId = ad_utility::testing::makeGetId(qec->getIndex());
+  using enum Datatype;
+
+  auto checkIdToLiteralOrIri =
```

Comment:
That lambda is useful, but add a small comment what it exactly does.

Reply:
Edit:
I see that you explain the test setup beelow, then add asmall dummy comment here that tells us that the explanation is below with the test cases.